stock_inventory_cost_info/models/stock_inventory.py

```diff
@@ -14,9 +14,9 @@ class InventoryLine(models.Model):
         string="Adjustment cost", compute="_compute_adjustment_cost", store=True
     )
 
-    @api.depends("product_qty", "theoretical_qty", "inventory_id.state")
+    @api.depends("difference_qty", "inventory_id.state")
     def _compute_adjustment_cost(self):
         for record in self:
-            adjusted_qty = record.product_qty - record.theoretical_qty
-            adjustment_cost = adjusted_qty * record.product_id.standard_price
-            record.adjustment_cost = adjustment_cost
+            record.adjustment_cost = (
+                record.difference_qty * record.product_id.standard_price
```

Comment:
Oups missing product_id.standard_price

Reply:
Not really, as we don't want to recompute all the past inventories when the cost price is changed. We only want to recompute it when validating or creating it.

Reply:
Ok, makes sense